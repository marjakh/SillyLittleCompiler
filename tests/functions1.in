function f() {
   write(2511);
}

write(1);
f();
write(2);
