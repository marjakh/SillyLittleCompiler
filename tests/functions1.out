1
2511
2
